### PR TITLE
Collision mapcheck

### PR DESCRIPTION
**Pull request by Frickerson**

### CODE REVIEW

File: Source/Private/Validators/MapCheckValidator_Collisions.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
+#include "Validators/MapCheckValidator_Collisions.h"
+
+#include "Misc/UObjectToken.h"
```
Comment (TheEmidee) [inline Source/Private/Validators/MapCheckValidator_Collisions.cpp]: <>

File: Source/Private/Validators/MapCheckValidator_Collisions.cpp [hunk 2]
```diff
@@ -0,0 +1,72 @@
+#include "Validators/MapCheckValidator_Collisions.h"
+
+#include "Misc/UObjectToken.h"
+
+#include <Engine/Light.h>
```
Comment (TheEmidee) [inline Source/Private/Validators/MapCheckValidator_Collisions.cpp]: is this needed?

**Review by TheEmidee — CHANGES_REQUESTED** [review]

File: Source/Private/Validators/MapCheckValidator_Collisions.cpp (hunk 2 above)
Comment (Frickerson) [reply Source/Private/Validators/MapCheckValidator_Collisions.cpp]: Removed.

File: Source/Private/Validators/MapCheckValidator_Collisions.cpp (hunk 1 above)
Comment (Frickerson) [reply Source/Private/Validators/MapCheckValidator_Collisions.cpp]: Done.

**Review by TheEmidee — APPROVED** [review]

**Pull request merged**